spec/index.html

```diff
@@ -396,6 +429,13 @@ <h3>Document Element</h3>
 &lt;/sparql&gt;
 </pre>
   <p class="note">Different values of <code>its:version</code> are allowed.</p>
+  <p>The optional <code>version</code> attribute MAY be used on the <code>sparql</code> element to announce that RDF 1.2 functionalities are used in the results as follows.</p>
```

Comment:
There is nothing wrong here but personally, I'd put in before the `its:version` because it is more significant. I would also be tempted to drop the  `<?xml version="1.0"?>` ) -- too many uses of "version" close together.